regulations/static/regulations/css/less/module/note.less

```diff
@@ -7,3 +7,25 @@
       margin-top: 0;
     }
 }
+.footnote-box {
+    background-color: @bg_gray;
+    padding: 1.5em;
+    margin-left: 20px;
+
+    h5 {
+      margin-top: 0;
+      font-family: "Lato Bold";
```

Comment:
@jbarnicle: I know you have experience with site font wrangling, care to comment on how this should be handled?